<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stkAuroraTrackerWidget</class>
 <widget class="QWidget" name="stkAuroraTrackerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stkAuroraTrackerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>串口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comPortChoiceComboBox">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>服务器IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverIPEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>服务器端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverPortEdit">
       <property name="text">
        <string>18944</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectServerButton">
       <property name="text">
        <string>连接服务器</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="openDeviceButton">
       <property name="text">
        <string>打开设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeDeviceButton">
       <property name="text">
        <string>关闭设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startTrackingButton">
       <property name="text">
        <string>开始跟踪</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopTrackingButton">
       <property name="text">
        <string>停止跟踪</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="toolsTableWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>147</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>147</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
